### PR TITLE
Moving mod/feedtest to src/Module/Feedtest

### PR DESCRIPTION
**Pull request by nupplaphil**
Part of #4090 

I had to add `Contact::getById()` because I didn't find any similar query for Contacts.

### CODE REVIEW

File: src/Module/Feedtest.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			$dummy = null;
+			$import_result = Protocol\Feed::import($xml, $importer, $contact, $dummy, true);
+
+			$result = [
+				'input' => $xml,
+				'output' => var_export($import_result, true),
+			];
+		}
+
+		print_r($result);
```
Comment (MrPetovan) [inline src/Module/Feedtest.php]: Please remove debug code.

File: src/Module/Feedtest.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	public static function content()
+	{
+		$result = [];
+		if (!empty($_REQUEST['url'])) {
+			$url = $_REQUEST['url'];
+
+			$importer = Model\User::getById(local_user());
+
+			$contact_id = Model\Contact::getIdForURL($url, local_user(), true);
+			$contact = Model\Contact::getById($contact_id);
```
Comment (annando) [inline src/Module/Feedtest.php]: I think that a simple ```DBA::selectFirst('contact', [], ['id' => $contact_id]);``` is enough here.

File: src/Model/Contact.php [hunk 3]
```diff
@@ -109,6 +109,16 @@ class Contact extends BaseObject
 	 * @}
 	 */
 
+	/**
+	 * @param  integer       $id
+	 * @return array|boolean Contact record if it exists, false otherwise
+	 * @throws \Exception
+	 */
+	public static function getById($id)
```
Comment (annando) [inline src/Model/Contact.php]: I would like to add such functions only when they contain some additional functionality (like the similar named function in the ```User``` class). And if we have the need for encapsulating some stuff, I would prefer the way like we did in the ```Item``` class. (Not the uglyness of the code there, but using the same function names like in the database class)

**Comment by nupplaphil** [comment]
The intention behind `Mode\Contact::getById()` was to eliminate the dependency to direct DB-calls (removing the `DBA` namespace). I thought that this tiny class would be a good place to add it without side effects or noise.

The `Item` class as mentioned is a Model, so the DBA calls are perfectly fit to use here. I see the Model as the "service" for other modules to serve the information of the datastore behind it. So f.e. if we ever need to restructure f.e. the `contact` table, we don't need to search through the code, it should have an impact straight just in the `Contact` model.

It's a separation of responsibility in this case. I read the basic intention at this place too https://github.com/friendica/friendica/issues/3878#issue-271901424 , so I thought it would fit.

**Pull request merged**